docs/source/installation/guide_quick/index.rst

```diff
@@ -0,0 +1,83 @@
+.. _installation:guide-quick:
+
+========================
+Quick installation guide
+========================
+
+.. attention::
+
+    AiiDA requires a recent version of Python.
+    Please refer to the `Python Package Index <https://pypi.org/project/aiida-core/>`_ for the minimum required version.
```

Comment:
Just for curiosity:
Is it possible to somehow make this automatic?  if python version is below 3.9, `aiida-core` refuses to be installed and raise an a proper message. Is it already like that from pip side?

Reply:
This is indeed done automatically, at least with `pip` as far as I know. We specify `python_requires>=3.9` in the `pyproject.toml` so as long as the install tool reads this metadata and errors when an older version is being used, it should work.